app/views/caseload.html

```diff
@@ -12,133 +12,219 @@
+              <th colspan="9" id="{{caseType.displayName}}_tiers">Case tiers</th>
+            </tr>
+            <tr>
+              <th id="{{caseType.displayName}}_name">Name</th>
+              <th id="{{caseType.displayName}}_grade">Grade</th>
+              <th id="{{caseType.displayName}}_overall">Overall</th>            
+              <th id="{{caseType.displayName}}_untiered">Untiered</th>
+              <th id="{{caseType.displayName}}_d2">D2</th>
+              <th id="{{caseType.displayName}}_d1">D1</th>
+              <th id="{{caseType.displayName}}_c2">C2</th>
+              <th id="{{caseType.displayName}}_c1">C1</th>
+              <th id="{{caseType.displayName}}_b2">B2</th>
+              <th id="{{caseType.displayName}}_b1">B1</th>
+              <th id="{{caseType.displayName}}_a">A</th>
+            </tr>
+          </thead>
+          <tbody class="">
+            {% for member in caseType.array %}
+            {% for i in range(0, member.grades.length) %}
+              <tr>
+                {% if i === 0 %}
+                <td rowspan="{{ member.grades.length }}" headers="{{caseType.displayName}}_om {{caseType.displayName}}_name"><a href="/{{ childOrganisationLevel }}/{{ member.linkId }}">{{ member.name }}</a></td>
+                {% endif %}
+                <td headers="{{caseType.displayName}}_om {{caseType.displayName}}_grade">{{ member.grades[i].grade }}</td>            
+                <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_overall" >{{ member.grades[i].totalCases | round(1) }}</td>              
+                <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_untiered">{{ member.grades[i].untiered | round(1) }}</td>
+                <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_d2">{{ member.grades[i].d2 | round(1) }}</td>
+                <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_d1">{{ member.grades[i].d1 | round(1) }}</td>
+                <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_c2">{{ member.grades[i].c2 | round(1) }}</td>
+                <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_c1">{{ member.grades[i].c1 | round(1) }}</td>
+                <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_b2">{{ member.grades[i].b2 | round(1) }}</td>
+                <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_b1">{{ member.grades[i].b1 | round(1) }}</td>
+                <td headers="{{caseType.displayName}}_tiers {{caseType.displayName}}_a">{{ member.grades[i].a | round(1) }}</td>
+              </tr>
+            {% endfor %}
+            {% endfor %}
+          </tbody>
+        </table>
+      </div>
+    {% endif %}
+    {% endfor %}
+  </div>
 
+  <div id="caseload-by-grade">
+    {% for caseType in caseloadDetails %}
+    {% if caseType.displayName === 'Overall' %}  
```

Comment:
Slightly confusing that this table sits below the tabs but doesn't update for Community, Custody, License. Although maybe that is what they want. Not sure.

Reply:
Agreed its confusing by its neater at the bottom as we decided